### PR TITLE
silution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
- [DEMO LINK](https://Zelinskyi-Serhii.github.io/layout_stop-watch/)
Solution*

### CODE REVIEW

File: src/styles/clock.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  &-arrows {
+    position: absolute;
+    bottom: 25px;
+    border-radius: 50%;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    transform-origin: bottom center;
+  }
```
Comment (kirill-vypirovskyi) [inline src/styles/clock.scss:30]: You can do it as %placeholder and don't add one more class to arrows

**Review by kirill-vypirovskyi — APPROVED** [review]

**Review by Ambulence — APPROVED** [review]
Good job!

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+
+    <div class="clock">
+      <div class="clock__minute clock-arrows"></div>
```
Comment (Andr1yK) [inline src/index.html:14]:
```suggestion
      <div class="clock__minute clock-arrow"></div>
```

File: src/styles/clock.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  &__minute {
+    left: 21px;
+    height: 150px;
+    width: 8px;
+    background-color: yellowgreen;
+    animation: round 3600s;
+  }
+
+  &-arrows {
```
Comment (Andr1yK) [inline src/styles/clock.scss:23]: wow.. why do u style block inside block?

**Review by Andr1yK — CHANGES_REQUESTED** [review]

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**